.github/workflows/main.yml

```diff
@@ -353,6 +353,30 @@ jobs:
         toolchain: ${{ matrix.rust }}
     - run: cargo test --all-features --workspace
 
+  test_msrv:
+    name: Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        build: [msrv]
+        include:
+          - build: msrv
+            os: ubuntu-latest
+            rust: 1.58
```

Comment:
In some other actions we scrape this value out of `Cargo.toml`...see e.g. https://github.com/coreos/bootupd/blob/e167ea9e7c2539654aba94f903061718ac499bc1/.github/workflows/rust.yml#L57

If you choose to keep this as is, probably worth adding a note in `Cargo.toml` to bump the `rust-version` there too.

Thanks for digging into this!

Reply:
Ah, I hadn't added a rust-version field yet. I've now added it, with a comment!